test/presenters/brexit_landing_page_presenter_test.rb

```diff
@@ -109,8 +109,8 @@
 
   describe "#email_path" do
     it "strips the locale extension from the base path if present" do
-      I18n.with_locale(:fr) do
-        taxon = Taxon.new(ContentItem.new("base_path" => "/base_path.fr"))
+      I18n.with_locale(:cy) do
```

Comment:
After changing from loading the campaign buckets as a YAML file to using I18n (https://github.com/alphagov/collections/pull/1346/commits/3ea3c92bfa71fb145acd1a8ac413bde4b04a0f3e#diff-e85c65799e3ed5ade04dee5e8ed37f93R54) this test broke as it wouldn't return an array since `fr` translation files don't exist in this repo, which meant that an error occurred when trying to iterate over it.

Reply:
Ah yes, I see. That should be fine - just trying to think if there's a new edge case there.